src/uiSelectMultipleDirective.js

```diff
@@ -212,6 +212,14 @@ uis.directive('uiSelectMultiple', ['uiSelectMinErr','$timeout', function(uiSelec
         $selectMultiple.updateModel();
       });
 
+      scope.$on('uis:select-multiple', function (event, items) {
+        if ($select.selected.length >= $select.limit) {
+          return;
+        }
+        $select.selected = $select.selected.concat(items);
```

Comment:
I think this broke some things without the onSelectCallback stuff above. Hold off for a second while I work on fixing it.

Reply:
On closer examination I don't think I broke anything. The onSelectCallback is only used in a few instances and no where that we can actually paste into. It could be an enhancement in the future, but for now I think this is fine.